packages/docs/content/13-release-notes.md

```diff
@@ -13,6 +13,30 @@ import InfoBox from '@librarium/shared/src/components/InfoBox';
 import PointsOfInterest from '@librarium/shared/src/components/common/PointOfInterest';
 import Tooltip from "@librarium/shared/src/components/ui/Tooltip";
 
+# January 07, 2021 - Release 1.7.0
+
+We are pleased to announce the release of Spectro Cloud Version 1.7.0 focusing on a dedicated User Interface which smoothens the migration from a quick start version to enterprise version.
```

Comment:
This release focuses on a dedicated User Interface which eases the migration from a quick start version to enterprise version. Brownfield support has been added with this release.